### ISSUE TITLE
Contact Us Data - User Linking Issue

### ISSUE THREAD

**Issue by NVombat** [NVombat/MusicApp#148]
### Description

When updating the contact us form, the table is updated however the user_profile/user_data should be updated with the contact_us query_id so that we can keep a track of the queries the user has raised. Additionally, statuses of Resolved and Unresolved should be added to give an idea of the progress of the query.

The contact_us collection and the user_data collection should be linked as well

### Expected behavior

Contact us data should show up on the user_profile with an ID

### Actual behavior

The contact us query does not get added to the user_profile

### Steps to reproduce

Each contact us query should be given an ID
When a query is raised, the ID should be appended to the User Profile ContactUs List
The query should have proper defined status

### Context

Improve the interlinking of methods and keep a track of all user activities. There should be a clear connection between the activities a user performs

**Comment by saeideh-eslamian** [comment]
Hi, May I start working on this issue?

**Comment by NVombat** [comment]
Yes, you can take this issue up. If you need any help let me know.
